includes/object-cache.php

```diff
@@ -708,6 +709,7 @@ protected function connect_using_phpredis( $parameters ) {
                     $parameters['scheme'],
                     str_replace( 'tls://', '', $parameters['host'] )
                 );
+                $args['ssl'] = $parameters['ssl'];
```

Comment:
This doesn't work with PhpRedis. As a general rule, we don't merge changes that haven't been tested thoroughly, because it puts too many sites at risk.